docs/pages/workflow/expo-cli.md

```diff
@@ -283,7 +283,7 @@ From here, you can choose to generate basic project files like:
 - `CI` (**boolean**) when enabled, the CLI will disable interactive functionality, skip optional prompts, and fail on non-optional prompts. Example: `CI=1 npx expo install --check` will fail if any installed packages are outdated.
 - `EXPO_NO_TELEMETRY` (**boolean**) disables anonymous usage collection.
 - `EXPO_NO_GIT_STATUS` (**boolean**) skips warning about git status during potentially dangerous actions like `npx expo prebuild --clean`.
-- `EXPO_ENABLE_INTERSTITIAL_PAGE` (**boolean**) enables the experimental "interstitial page" for selecting an app, that shows when a user has `expo-dev-client` installed, and starts the project with `expo start` instead of `expo start --dev-client`.
+- `EXPO_NO_REDIRECT_PAGE` (**boolean**) disables the redirect page for selecting an app, that shows when a user has `expo-dev-client` installed, and starts the project with `expo start` instead of `expo start --dev-client`.
```

Comment:
```suggestion
- `EXPO_NO_REDIRECT_PAGE` (**boolean**) disables the redirect page for selecting an app, that shows when a user has `expo-dev-client` installed, and starts the project with `npx expo start` instead of `npx expo start --dev-client`.
```

I think the commands here should start with `npx` prefix.

Reply:
nice catch, thanks!